src/py42/services/preservationdata.py

```diff
@@ -19,8 +19,8 @@ def find_file_version(self, file_md5, file_sha256, paths):
         uri = u"/api/v1/FindAvailableVersion"
         return self._connection.post(uri, json=data)
 
-    def get_file_version_list(self, deviceGuid, file_md5, file_sha256, path):
-        params = u"fileSHA256={}&fileMD5={}&deviceGuid={}&path={}"
-        params = params.format(file_sha256, file_md5, deviceGuid, quote(path))
-        uri = u"/api/v1/FileVersionListing?{}".format(params)
+    def get_file_version_list(self, deviceUid, file_md5, file_sha256, path):
+        params = u"fileSHA256={}&fileMD5={}&deviceUid={}&filePath={}"
```

Comment:
Why not just use the `params=dict` in the call to `get()` rather than build your own query string?

Reply:
I just copied this from what the old endpoint did, but yeah, that would be more consistent with what we have else where.

Reply:
Nevermind - I remember there is some annoying reason for this

Reply:
yeah i remembered too, its the encoding on `filePath`.